app/components/repo-status-badge.js

```diff
@@ -0,0 +1,30 @@
+import Component from '@ember/component';
+import { inject as service } from '@ember/service';
+import { computed } from '@ember/object';
+import { reads } from '@ember/object/computed';
+
+export default Component.extend({
+  tagName: '',
+  statusImages: service(),
+
+  repo: null,
+  onClick() {},
+
+  lastBuildState: reads('repo.defaultBranch.lastBuild.state'),
+  defaultBranch: reads('repo.defaultBranch.name'),
+
+  statusImageUrl: computed(
+    'repo.slug',
+    'repo.private',
+    'defaultBranch',
+    'lastBuildState',
+    function () {
+      const { defaultBranch, lastBuildState = 'unknown', repo } = this;
+
+      const url = this.statusImages.imageUrl(repo, defaultBranch);
+      const divider = (url.indexOf('?') >= 0) ? '&' : '?';
```

Comment:
I think it's pretty safe to use
```suggestion
      const divider = url.includes('?') ? '&' : '?';
```
What do you think?

Reply:
Ahh yeah, that sounds good to me :)